packages/api/models/Error.ts

```diff
@@ -9,6 +9,5 @@ export interface Error {
 }
 
 export interface ValidationError extends Error {
-  code: ErrorCode.INVALID_INPUT;
```

Comment:
I think we want to keep `ErrorCode.INVALID_INPUT` correct? Because that enforces that ValidationErrors always have `INVALID_INPUT` for the `code`?

Reply:
I think I simply made a mistake attempting conflict resolution.